app/logout.go

```diff
@@ -0,0 +1,38 @@
+package app
+
+import (
+	"fmt"
+	"github.com/urfave/cli/v2"
+)
+
+func NewLogoutCommand(c *LoginClient) (CommandOut, error) {
+	return CommandOut{Command: &cli.Command{
+		Name:    "logout",
+		Usage:   "Logout current user",
+		Aliases: []string{"lo"},
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "domain",
+				Value:    "login.tmprl.cloud",
+				Aliases:  []string{"d"},
+				Required: false,
+				Hidden:   true,
+			},
+			&cli.BoolFlag{
+				Name:     "disable-pop-up",
+				Usage:    "disable browser pop-up",
+				Required: false,
+			},
+		},
+		Action: func(ctx *cli.Context) error {
+			logoutURL := fmt.Sprintf("https://%s/v2/logout", ctx.String("domain"))
+			fmt.Printf("Logout via this url: %s\n", logoutURL)
+			if !ctx.Bool("disable-pop-up") {
```

Comment:
does it ever make sense to pop up?

Reply:
Yes, i believe pop-up is a good default behavior so you don't have to click on a link